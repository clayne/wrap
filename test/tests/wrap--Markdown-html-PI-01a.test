wrap | /dev/null | -u | md-html-PI-01a.md | 0
