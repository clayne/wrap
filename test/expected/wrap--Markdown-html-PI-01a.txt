This is some text.  This is some more text.
<?This is a processing instruction.?>
This is some text.  This is some more text.
